editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx

```diff
@@ -12,36 +12,41 @@ import QuestionHeader from "@planx/components/shared/Preview/QuestionHeader";
 import { PrivateFileUpload } from "@planx/components/shared/PrivateFileUpload/PrivateFileUpload";
 import type { PublicProps } from "@planx/components/ui";
 import buffer from "@turf/buffer";
-import { type GeometryObject, point } from "@turf/helpers";
+import { type Feature, point } from "@turf/helpers";
 import { Store, useStore } from "pages/FlowEditor/lib/store";
 import React, { useEffect, useRef, useState } from "react";
 import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 import FullWidthWrapper from "ui/public/FullWidthWrapper";
 
-import { DrawBoundary, PASSPORT_UPLOAD_KEY } from "../model";
+import { DrawBoundary, PASSPORT_UPLOAD_KEY, UserDrawAction } from "../model";
 
 export type Props = PublicProps<DrawBoundary>;
 
-export type Boundary = GeometryObject | undefined;
+export type Boundary = Feature | undefined;
 
 // Buffer applied to the address point to clip this map extent
 //   and applied to the site boundary and written to the passport to later clip the map extent in overview documents
-const BUFFER_IN_METERS = 75;
+const BUFFER_IN_METERS = 100;
```

Comment:
This better accomodates large default boundaries - I think this is still a reasonable distance for our OS licensing concerns?